ecosystem/sep-0024.md

```diff
@@ -190,6 +192,127 @@ The deposit flow can only be fulfilled if the Stellar `Account` has established
 5. `Anchor` detects the trust line creation in the `Account`. If the asset is `AUTH_REQUIRED`, `Anchor` approves the new trust line.
 6. `Anchor` proceeds with the deposit flow.
 
+##### Claimable Balances
```

Comment:
Can you start with stating that claimable balances are currently optional by highly recommended? Also, note that in the future, claimable balances will be required.

Eventually, the hope is that all anchors will support CB's, and therefore we no longer need to support the 'waiting for trust-line' flow.

Reply:
How does the description read now?